cmd/storage-rest-server.go

```diff
@@ -525,6 +525,48 @@ func (s *storageRESTServer) DeleteFileBulkHandler(w http.ResponseWriter, r *http
 	w.(http.Flusher).Flush()
 }
 
+// DeletePrefixesErrsResp - collection of delete errors
+// for bulk prefixes deletes
+type DeletePrefixesErrsResp struct {
+	Errs []error
+}
+
+// DeletePrefixesHandler - delete a set of a prefixes.
+func (s *storageRESTServer) DeletePrefixesHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This could take a long time, you need to keep the connection alive.

You can copy the existing code (which is rather clunky) or wait a little for this: https://github.com/minio/minio/pull/9075/files#diff-9c5604c6beb0f12c48e5615563d70c97R557

Reply:
Yes the DeleteFileBulk() code has to be copied here..

Reply:
> Yes the DeleteFileBulk() code has to be copied here..


Done

Reply:
Actually it is not done in delete file bulk, I just added it in this PR

Reply:
:+1: